odh/base/monitoring/github-receiver.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: github-receiver
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: github-receiver
+  template:
+    metadata:
+      labels:
+        run: github-receiver
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: '9990'
+    spec:
+      containers:
+        - name: github-receiver
+          image: quay.io/4n4nd/alertmanager-github-receiver:v0.10
+          env:
+            - name: GITHUB_AUTH_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: github-secrets
```

Comment:
Do we have the secret available anywhere? I'm missing it from the manifests here.

Reply:
I haven't added the secret yet. Thought I can update it once we actually start setting up all our Prometheus alerts?

Reply:
I don't think we can upload this secret, we don't have a secret management solution set up yet -- this needs to definitely be a to-do and come with an issue attached, for now we'll just have to settle with having to manually deploy the secret before deploying it to prod. 

We could just upload the secret encrypted with some gpg key (we haven't made one yet for operate-first, though we should) and not include in the kustomization.

Reply:
let's use @tumido's suggestion [here](https://github.com/operate-first/apps/pull/72#discussion_r539357449) and add a dummy/stub secret with this and not include it in the kustomization.yaml

Reply:
I'm fine with that, but we should have an issue attached to this so that we don't forget about it when we go back and add all our secrets.

Reply:
Let's get a sops yaml and ksops encrypted secret instead as per our offline discussion, but not include it in the kustomization just yet. And link this secret [here](https://github.com/operate-first/apps/issues/77)

Reply:
added the ksops encrypted secret: https://github.com/operate-first/apps/pull/68/commits/f88928116696bd0d12554fe7d10425ef71f7e07e